selfdrive/car/gm/values.py

```diff
@@ -99,6 +100,7 @@ def init_make(self, CP: car.CarParams):
   CAR.MALIBU: GMCarInfo("Chevrolet Malibu Premier 2017"),
   CAR.ACADIA: GMCarInfo("GMC Acadia 2018", video_link="https://www.youtube.com/watch?v=0ZN6DdsBUZo"),
   CAR.BUICK_REGAL: GMCarInfo("Buick Regal Essence 2018"),
+  CAR.ESCALADE: GMCarInfo("Cadillac Escalade 2017", "Adaptive Cruise Control (ACC) & LKAS"),
```

Comment:
```suggestion
  CAR.ESCALADE: GMCarInfo("Cadillac Escalade 2017", "Driver Assist Package"),
```

https://cdn.dealereprocess.org/cdn/brochures/cadillac/2017-escalade.pdf